pkg/common/map.go

```diff
@@ -33,3 +35,45 @@ func MergeMaps(baseMap map[string]interface{}, extraMaps ...map[string]interface
 
 	return baseMap
 }
+
+// MergeStringMaps - merge two or more string->map maps
+func MergeStringMaps(baseMap map[string]string, extraMaps ...map[string]string) map[string]string {
+	InitMap(&baseMap)
+
+	for _, extraMap := range extraMaps {
+		for key, value := range extraMap {
+			baseMap[key] = value
```

Comment:
this looks like Joining (with overriding existing keys), not merging?

Reply:
it merges/joins, but yes, if the key already exists, it will be overwritten